### PR TITLE
Added Mountain Madness 2021 and Fall Hacks 2020 pages

### PR DESCRIPTION
**Pull request by SpartanApple**

### CODE REVIEW

**Review by modernNeo — CHANGES_REQUESTED** [review]
why is a `.swp` file being commited?

File: csss-site/src/events/templates/events/fall_hacks2020.html [hunk 1]
```diff
@@ -0,0 +1,334 @@
+<!DOCTYPE html>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/fall_hacks2020.html:1]: any particular reason this isn't following the template design of our website?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/csss/templates/csss/new_header.html [hunk 2]
```diff
@@ -97,6 +97,8 @@
             <a href="{{URL_ROOT}}events/board_games" class="navbar-item">What are Board Games Night?</a>            
             <a href='{{ URL_ROOT}}events/frosh/' class="navbar-item">CSSS FroshWeek</a>
             <a href="{{URL_ROOT}}events/mountain_madness2020" class="navbar-item">Mountain Madness 2020</a>          
+            <a href="{{URL_ROOT}}events/mountain_madness2021" class="navbar-item">Mountain Madness 2021</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/new_header.html:100]: can you please order these in reverse so that its chronological order?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks2020.html (hunk 1 above)
Comment (SpartanApple) [reply csss-site/src/events/templates/events/fall_hacks2020.html:1]: No.

File: csss-site/src/csss/templates/csss/new_header.html [hunk 3]
```diff
@@ -97,6 +97,8 @@
             <a href="{{URL_ROOT}}events/board_games" class="navbar-item">What are Board Games Night?</a>            
             <a href='{{ URL_ROOT}}events/frosh/' class="navbar-item">CSSS FroshWeek</a>
             <a href="{{URL_ROOT}}events/mountain_madness2020" class="navbar-item">Mountain Madness 2020</a>          
+            <a href="{{URL_ROOT}}events/mountain_madness2021" class="navbar-item">Mountain Madness 2021</a>          
+            <a href="{{URL_ROOT}}events/fall_hacks2020" class="navbar-item">Fall Hacks 2020 Projects</a>
```
Comment (modernNeo) [inline csss-site/src/csss/templates/csss/new_header.html:101]: Please create a separate page for the Fall Hacks 2020.

and then in that page, just have a link to the submissions.

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/templates/events/fall_hacks2020.html (hunk 1 above)
Comment (modernNeo) [reply csss-site/src/events/templates/events/fall_hacks2020.html:1]: then please follow the template design of the website

**Comment by SpartanApple** [comment]
Will make appropriate changes.

**Pull request closed without merging**